Firmware/ultralcd.cpp

```diff
@@ -8750,21 +8752,31 @@ uint8_t get_message_level()
 
 void menu_lcd_longpress_func(void)
 {
+    lcd_longpress_trigger = 0;
 	backlight_wake();
-    if (homing_flag || mesh_bed_leveling_flag || menu_menu == lcd_babystep_z || menu_menu == lcd_move_z)
+    if ( (homing_flag || mesh_bed_leveling_flag) && (menu_menu == lcd_babystep_z || menu_menu == lcd_move_z))
     {
         // disable longpress during re-entry, while homing or calibration
+        MYSERIAL.println("disable longpress during re-entry, while homing or calibration");
         lcd_quick_feedback();
         return;
     }
     if (menu_menu == lcd_hw_setup_menu)
     {
+        MYSERIAL.println("menu_menu == lcd_hw_setup_menu");
```

Comment:
Use `puts_P(PSTR("The message"));`
It does the same thing but it keeps things in progmem instead of valuable ram. You might soon overflow the stack in certain situations

Reply:
Those prints were not supposed to be in the commit 👍 But good to know